config/environments/profile.rb

```diff
@@ -0,0 +1,41 @@
+# an environment configured for performance profiling.
```

Comment:
Hm. It looks like serve_static_files is [deprecated](http://edgeguides.rubyonrails.org/5_0_release_notes.html#railties-deprecations), and we would use [public_file_server.enabled](http://guides.rubyonrails.org/configuring.html#rails-general-configuration) instead. Default is true except in production environment. I think we need it to stay true since we aren't using an independent web server in this context. so I won't do anything special with that one.